### PR TITLE
Update README.md with version matrix

### LINKED ISSUE #159: Version matrix in README
## Problem

It isn't immediately obvious which version of celestia-app and celestia-node are running on which testnet

## Proposal

It may be helpful to include a version matrix in the README. Something like

Network | celestia-app | celestia-node
--- | --- | ---
Mamaki | https://github.com/celestiaorg/celestia-app/releases/tag/v0.6.0 | ...
Arabica | https://github.com/celestiaorg/celestia-app/releases/tag/v0.7.0 | ...

### PR DESCRIPTION
**Pull request by Bidon15**
## Overview

Closes #159 

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,7 @@
 
 This repository contains the `genesis.json` for Celestia testnets.
 
+If you want to see versions per testnets that are being used, then navigate to this [grafana dashboard](https://celestia.grafana.net/goto/JktIO8zSg?orgId=1)
```
Comment (rootulp) [inline README.md:5]: [question] is this Grafana dashboard publicly visible? When I navigate to it and login via a non Celestia org account, I observe this error:

Screenshot 2023-09-06 at 2 40 46 PM

If this dashboard isn't publicly accessible, I think we should maintain a publicly accessible version matrix. In other words, I don't think we should close https://github.com/celestiaorg/networks/issues/159 yet.

btw the dashboard is nice, and I'll keep it bookmarked 👍

**Review by rootulp — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Bidon15) [inline README.md:5]:
```suggestion
If you want to see versions per testnets that are being used, then navigate to this [grafana dashboard](https://celestia.grafana.net/public-dashboards/5d14d96e44f04664bb0c44267e5d645c)
```

Comment (Bidon15) [reply README.md]: I've changed the link to a public view, so there is no login needed. 
Thanks for the catch! Should be fine now

**Review by jrmanes — APPROVED** [review]
LGTM! 🚀

**Pull request merged**